src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -19,14 +18,21 @@ import {withNetwork} from '../../../components/OnyxProvider';
 import CONST from '../../../CONST';
 import applyStrikethrough from '../../../components/HTMLEngineProvider/applyStrikethrough';
 import editedLabelStyles from '../../../styles/editedLabelStyles';
+import UserDetailsTooltip from '../../../components/UserDetailsTooltip/UserDetailsTooltip';
 
 const propTypes = {
+    /** Source image */
+    avatarSource: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
+
+    /** User's name */
+    name: PropTypes.string,
+
+    /** User's login */
+    login: PropTypes.string,
+
```

Comment:
Why did you add these? Did you plan to use them for fallback?

Reply:
I missed these in the refactor, fixing it now